pandas/io/formats/format.py

```diff
@@ -1621,18 +1621,23 @@ def __init__(
 
     def _format_strings(self) -> list[str]:
         """we by definition have DO NOT have a TZ"""
-        values = self.values
```

Comment:
what is the scenario in which we get here with non-1D values?  could we just assert 1d-ness at the top and not worry about the rest?